universal-application-tool-0.0.1/app/models/Applicant.java

```diff
@@ -63,4 +69,8 @@ public Account getAccount() {
   public void setAccount(Account account) {
     this.account = account;
   }
+
+  public ImmutableList<Application> getApplications() {
```

Comment:
question (non-blocking): Would it make sense to add a test for this? To make sure the mapping is how we expect it.

Reply:
Definitely, added.